app/controllers/homepage_controller.rb

```diff
@@ -19,7 +19,7 @@ def search_component
   helper_method :search_component
 
   def publication_class
-    HomepagePresenter
```

Comment:
I don't think we should be putting this into publication class. Can we not update the views so that they don't use publication (since this is the content item, they can use that).

Reply:
That's a good shout. It turns out that only the popular links partial was calling publication and the homepage doesn't need to rely on it for a breadcrumb.

I've pushed up another commit to remove the `publication_class`

Reply:
Nice